### PR TITLE
Updated Navigation Command in VS Code Cheatsheet

### PR DESCRIPTION
**Pull request by Karmpatel960**
Added Some Shortcuts in Integrated Terminal

### Checklist:
- [x] Read our [contributing guidelines](../docs/CONTRIBUTING.md).
- [x] Search for duplicates.
- [x] Used an informative name for this pull request.

### Follow-up
- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by crescentpartha — CHANGES_REQUESTED** [review]
Please, modify according to the requirement.

File: CheatSheets/vscode-cheatsheet.md [hunk 1]
```diff
@@ -130,5 +130,9 @@
 | ------- | ----------- |
 | `` Ctrl + ` `` | Show integrated terminal |
 | `` Ctrl + Shift + ` `` | Create new integrated terminal |
+| `` Ctrl + C ``| Copy selection |
+| `` Ctrl + V `` | Paste into active terminal |
+| `` Ctrl + ↑/↓ ``| Scroll up/down |
+| `` Ctrl+ Home/End ``| Scroll top/bottom |
```
[the remark this answers is not in the document]
Comment (Karmpatel960) [reply CheatSheets/vscode-cheatsheet.md]: Changes Done Now you can check it,

File: CheatSheets/vscode-cheatsheet.md [hunk 2]
```diff
@@ -130,5 +130,9 @@
 | ------- | ----------- |
 | `` Ctrl + ` `` | Show integrated terminal |
 | `` Ctrl + Shift + ` `` | Create new integrated terminal |
+| `` Ctrl + C ``| Copy selection |
+| `` Ctrl + V `` | Paste into active terminal |
```
[the remark this answers is not in the document]
Comment (Karmpatel960) [reply CheatSheets/vscode-cheatsheet.md]: Changes done Delete the commands

**Comment by Karmpatel960** [comment]
Hey @crescentpartha  I have Done Changes as you said please review it.

**Comment by Karmpatel960** [comment]
Hey @crescentpartha I done Changes as you mention in it Please Review it.

**Review by crescentpartha — APPROVED** [review]
Approved your changes.

**Pull request merged**